part-2-server/server.js

```diff
@@ -0,0 +1,102 @@
+var http = require('http');
+var fs = require('fs');
+//var readfiles = require("./createPoemFiles.js")
+//var i = request.url
+//console.log(request.url)
+
+var server = http.createServer(function(request, response){
+	var url = request.url;
+	if(url==='/'){
+		fs.readFile('index.html', function(err,data){
+			response.write(data.toString());
+			response.end();
+		})
+	}
+	else if(url === "/page-2" || url === "/page-3"){
+		fs.readFile('page-2.html', function(err,data){
+			response.write(data.toString());
+			response.end();
+		})
+	}
+
```

Comment:
I know it wouldn't work for the rest. I know I had to use a variable for
number and a for loop but couldn't implement it in the code.

On Sep 9, 2016 1:48 PM, "nmadd" <notifications@github.com> wrote:

> In part-2-server/server.js
> <https://github.com/C4Q/ac3.1-exam-4/pull/24#discussion_r78220804>:
>
> > +
> > +var server = http.createServer(function(request, response){
> > +	var url = request.url;
> > +	if(url==='/'){
> > +		fs.readFile('index.html', function(err,data){
> > +			response.write(data.toString());
> > +			response.end();
> > +		})
> > +	}
> > +	else if(url === "/page-2" || url === "/page-3"){
> > +		fs.readFile('page-2.html', function(err,data){
> > +			response.write(data.toString());
> > +			response.end();
> > +		})
> > +	}
> > +
>
> This will work for poems 0-10 but what about all the rest?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/C4Q/ac3.1-exam-4/pull/24/files/7324672043529e8b01f70cff04b857d0c167e431#r78220804>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATauGH9e4yEpmd9pzrexQpbFT6o5sTknks5qoZwFgaJpZM4J4crl>
> .
>